pandas/core/frame.py

```diff
@@ -3501,27 +3501,37 @@ def notnull(self):
     def dropna(self, axis=0, how='any', thresh=None, subset=None,
                inplace=False):
         """
+        Remove missing values.
+
         Return object with labels on given axis omitted where alternately any
-        or all of the data are missing
+        or all of the data are missing.
 
         Parameters
         ----------
         axis : {0 or 'index', 1 or 'columns'}, or tuple/list thereof
-            Pass tuple or list to drop on multiple axes
-        how : {'any', 'all'}
-            * any : if any NA values are present, drop that label
-            * all : if all values are NA, drop that label
+            Pass tuple or list to drop on multiple axes.
+        how : {'any', 'all'}, default 'any'
+            Determine if label is removed when we have at least one NA or all
+            NA.
+            * any : If any NA values are present, drop that label.
```

Comment:
can you put quotes around any (and all below) like `'any'` (to clearly indicate this are string values to pass)